src/main/java/org/opentripplanner/graph_builder/module/osm/GermanyWayPropertySetSource.java

```diff
@@ -69,8 +69,8 @@ public void populateProperties(WayPropertySet props) {
 
         /** We assume highway/cycleway of a cycle network to be safer (for bicycle network relations, their network is copied to way in postLoad */
         props.setProperties("lcn=yes", StreetTraversalPermission.ALL, 0.7, 0.7, true); // local cycle network
-        props.setProperties("rcn=yes", StreetTraversalPermission.ALL, 0.5, 0.5, true); // regional cycle network
-        props.setProperties("ncn=yes", StreetTraversalPermission.ALL, 0.9, 0.9, true); // national cycle network
+        props.setProperties("rcn=yes", StreetTraversalPermission.ALL, 0.7, 0.7, true); // regional cycle network
```

Comment:
I'm ok with these. In the long run, these scaling factors should be evaluated in an OR-condition, currently they are multiplied each and every one, resulting in a very heavy preference, when all three networks run over the same way, which is not intended.